manifests/init.pp

```diff
@@ -34,13 +34,13 @@
 #                               type:Optional[Stdlib::Absolutepath]
 #
 # $forward::                    The forward option
-#                               type:Optional[String]
+#                               type:Optional[Enum['only', 'first']]
```

Comment:
I'm not sure, I've never tested them without quotes. But since they are strings I would expect them to be quoted.